### PR TITLE
connect landing page

### PR DESCRIPTION
**Pull request by easyrun42**
## PR-Codex overview
This PR adds new components and updates existing ones related to the Connect feature. It introduces a `ConnectPlaygroundTab`, `ConnectPlaygroundButton`, and `PlaygroundMenu` for enhanced user interaction.

### Detailed summary
- Added `ConnectPlaygroundTab` component
- Introduced `ConnectPlaygroundButton` for language selection
- Included `PlaygroundMenu` component for interactive menu
- Updated styles and functionality for a better user experience

> The following files were skipped due to too many changes: `components/wallets/ConnectWalletPlayground/WalletButton.tsx`, `components/wallets/ConnectWalletMiniPlayground/MiniPlayground.tsx`, `components/connect/Carousel.tsx`, `components/connect/CodePlayground.tsx`, `pages/connect.tsx`, `public/assets/product-pages/connect/membership-overlay.svg`, `components/product-pages/common/connect/data.ts`, `public/assets/product-pages/connect/connect-lottie.json`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: contract-ui/tabs/code/components/code-overview.tsx [hunk 1]
```diff
@@ -627,11 +627,11 @@ export const CodeOverview: React.FC<CodeOverviewProps> = ({
             </Text>
           </Flex>
         </Flex>
-        {!onlyInstall ? (
+        {true ? (
           <>
-            {foundExtensions.length > 0 ? (
+            {true ? (
```
Comment (nachoiacovino) [inline contract-ui/tabs/code/components/code-overview.tsx:632]: can we simplify this? there's unreachable code here

File: components/connect/OverviewSection.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          base: "auto",
+          xl: `translateY(${-parallaxOffset + 200}px)`,
+        }}
+        transition="transform 0.8s cubic-bezier(0.33, 1, 0.68, 1)"
+      />
+    </Flex>
+  );
+};
+
+export default OverviewSection;
```
Comment (nachoiacovino) [inline components/connect/OverviewSection.tsx:68]: can we not export default for components

Comment (nachoiacovino) [reply components/connect/OverviewSection.tsx:68]: (there's a couple that do this in this PR)

File: components/connect/ConnectPlaygroundButton.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  },
+  react: {
+    icon: SiReact,
+    fill: "#61dafb",
+  },
+  "react-native": {
+    icon: SiReact,
+    fill: "#61dafb",
+  },
+  python: {
```
Comment (nachoiacovino) [inline components/connect/ConnectPlaygroundButton.tsx:24]: we shouldn't' show Python/Go/Unity

File: components/connect/Carousel.tsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    setSelectedShowCaseIdx((idx) => (idx === 2 ? 0 : idx + 1));
+  };
+
+  useEffect(() => {
+    if (!hoveredCard) {
+      const timer = setInterval(increment, 3500);
+
+      return () => clearInterval(timer);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (nachoiacovino) [inline components/connect/Carousel.tsx:55]: why do we need this? can we avoid this somehow please?

**Review by nachoiacovino — COMMENTED** [review]
left some comments but the main question here is that it seems like there's a lot of copy/pasted code from the dashboard side in which we do this programmatically?

It might be fine for now but it did seemed a bit weird to me

File: components/connect/OverviewSection.tsx (hunk 2 above)
Comment (easyrun42) [reply components/connect/OverviewSection.tsx:68]: Yep will clean up

File: contract-ui/tabs/code/components/code-overview.tsx (hunk 1 above)
Comment (easyrun42) [reply contract-ui/tabs/code/components/code-overview.tsx]: Ah yeah my bad, was testing and committed accidently

File: components/connect/Carousel.tsx (hunk 4 above)
Comment (easyrun42) [reply components/connect/Carousel.tsx:55]: Yep

**Comment by easyrun42** [comment]
> left some comments but the main question here is that it seems like there's a lot of copy/pasted code from the dashboard side in which we do this programmatically?
> 
> It might be fine for now but it did seemed a bit weird to me

Agree a lot of copy paste changes has happened here, but we're going back and fourth with this PR and deleting a lot of custom made components so once we're fully ready i'll do cleanup for that day.

**Pull request merged**